package.json

```diff
@@ -138,7 +138,7 @@
     "prettier": "1.16.4",
     "raf": "3.4.0",
     "raw-loader": "0.5.1",
-    "react": "16.8.2",
+    "react": "16.8.4",
```

Comment:
Interesting that this fell out of sync with `react-dom` below; they should always be the same